Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline
             containerTemplate
             {
                 name 'kiso-build-env'
-                image 'eclipse/kiso-build-env:v0.0.1'
+                image 'eclipse/kiso-build-env:latest'
```

Comment:
Makes sense - no Jenkinsfile update (pull request) on dockerfile change

Reply:
Indeed, however, this isn't without its downsides. We lose reproducibility of old jobs, as we cannot determine what image was used to build a particular commit. I weight that higher than the effort of updating this one-liner each time a new image is created.

I will therefore change it back to hard-versioned releases.